src/Util.java

```diff
@@ -0,0 +1,96 @@
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Util {
+
+	/**
+	 * Create utility method, which counts even numbers in given collection;
+	 */
+	public int countEvenNumbers(List<? extends Number> list){
+		int count = 0;
+		for (Number number: list) {
+			if(number.intValue() % 2 == 0)
```

Comment:
What is the result of `Util.countEvenNumbers(Arrays.asList(4.1, 4.2, 4.3)).countEvenNumbers()`?

Reply:
Result is 3... What it should be like?

Reply:
Probably the task isn't 100% clear, but I'd count  integer 4,  double 4.0, big decimal 4.00 or 4.(0) as an even numbers and 4.3 as an odd (or at least not even, because it's not an integer).
If I had to talk in the "code language", then: `number.doubleValue() % 2 == 0` does what I want.
Nevertheless, you've used bounded wildcard here in a way I expected you to use, and it was the primary goal, so technically task is correct.

Reply:
i think the same, i have already guessed what you mean)